src/services/TagsService.js

```diff
@@ -150,12 +150,12 @@ export default class TagsService {
    *
    * @return {boolean}
    */
-  containsKeyword(str) {
+  containsAnyKeyword(str) {
     if (_.isArray(str)) {
       let isInArray = false;
 
       str.forEach(ele => {
-        if (this.containsKeyword(ele)) {
+        if (this.containsAnyKeyword(ele)) {
           isInArray = true;
```

Comment:
I don't really understand what the result we should achieve ? In this code if I give array of words where last word will be not contains keyword all method return false. Is it correct ? Example:` ['test', 'tes', 'any']` with keyword 'tes' what should be expected result of call `true` or `false` ?